styles/pages/workflow.less

```diff
@@ -121,11 +121,11 @@
 	}
 }
 
-.pubids_export_actions {
+.export_actions {
 	padding-left: @double;
 	padding-right: @double;
 
-	.pubids_export_action {
+	.export_action {
```

Comment:
Hmm, we're going to have trouble modifying the .less files unless we cause the CSS cache to be removed on upgrade. (The CSS isn't recompiled unless the cache is manually deleted.) Mind adding an upgrade task to remove the cached CSS?